website/content/en/docs/reference/vrl/examples.md

```diff
@@ -8,9 +8,19 @@ Here you'll find a comprehensive list of all VRL program examples. These
 examples demonstrate the breadth of the language and its observability-focused
 facilities.
 
-## Try Using The REPL
+## Try Using the VRL subcommand
 
-You can run these examples using the **VRL REPL** (Read–eval–print loop).
+You can run these examples using the `vector vrl` subcommand with `--input` (single-line JSON file
+representing the incoming event) and `--program` (VRL program) to pass in the example input and
+program as well as `--print-object` to show the modified object.
```

Comment:
Technically it accepts multi-line files, as long as each line is a complete json object.

Reply:
Sure, but the examples are all single objects. I was just trying to call out that they should collapse them into one line before running.

Reply:
Yeah, that's fair as well 👍

Reply:
I pushed 8b7eb068c to clarify it a bit more.